transports/webrtc/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "libp2p-webrtc"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+description = "WebRTC transport for libp2p"
+repository = "https://github.com/libp2p/rust-libp2p"
+license = "MIT"
+edition = "2021"
+keywords = ["peer-to-peer", "libp2p", "networking"]
+categories = ["network-programming", "asynchronous"]
+
+[dependencies]
+asynchronous-codec = "0.6"
+async-trait = "0.1"
+bytes = "1"
+fnv = "1.0"
+futures = "0.3"
+futures-lite = "1"
+futures-timer = "3"
+hex = "0.4"
+if-watch = "2.0"
+libp2p-core = { version = "0.37.0", path = "../../core", default-features = false  }
+libp2p-noise = { version = "0.40.0", path = "../../transports/noise" }
+log = "0.4"
+multihash = { version = "0.16", default-features = false, features = ["sha2"] }
+multibase = "0.9"
+prost = "0.11"
+prost-codec = { version = "0.2", path = "../../misc/prost-codec" }
+rand = "0.8"
+rcgen = "0.9.3"
+serde = { version = "1.0", features = ["derive"] }
+stun = "0.4"
+thiserror = "1"
+tinytemplate = "1.2"
+tokio-crate = { package = "tokio", version = "1.18", features = ["net"]}
```

Comment:
We could probably do without the rename. This is usually only done when you need to mess with features and now that we have the new `dep:` syntax, I think the renames are no needed at all.